app/models/iiif_canvas_presenter.rb

```diff
@@ -46,6 +46,15 @@ def sequence_rendering
     }]
   end
 
+  def placeholder_content
+    # height and width from /models/master_file/extract_still method
+    IIIFManifest::V3::DisplayContent.new( @master_file.poster_master_file_url(@master_file.id),
+                                          width: 1280,
+                                          height: 720,
```

Comment:
Got the poster height and width from, https://github.com/avalonmediasystem/avalon/blob/main/app/models/master_file.rb#L378.